recipes/kraken/meta.yaml

```diff
@@ -23,11 +23,20 @@ build:
 requirements:
   build:
     - perl
-    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
 
+  host:
+    - perl
+    - clangdev  # [osx]
+    - llvm-openmp  # [osx]
   run:
     - jellyfish 1.*
```

Comment:
This probably needs to be `bioconda::jellyfish` as there is a conflicting package in conda-forge.

Reply:
I've changed it to `bioconda::jellyfish 1.*`